zypp/target/TargetImpl.cc

```diff
@@ -1541,13 +1541,9 @@ namespace zypp
           {
             DBG << "dryRun/downloadOnly: Not installing/deleting anything." << endl;
             if ( explicitDryRun ) {
-              if ( policy_r.singleTransModeEnabled() ) {
-                // single trans mode does a test install via rpm
-                commitInSingleTransaction( policy_r, packageCache, result );
-              } else {
-                // if cache is preloaded, check for file conflicts
-                commitFindFileConflicts( policy_r, result );
-              }
```

Comment:
Maybe we can even leave this in. If everything was downloaded, singletrans can check it as well.

Reply:
Yes. It's the case where the download-all either succeeded or had nothing to do.